meinberlin/assets/js/app.js

```diff
@@ -43,6 +43,9 @@ var dynamicFields = require('../../apps/contrib/assets/dynamic_fields.js')
 var getCurrentHref = function () {
   return location.href
 }
+var getCurrentPath = function () {
+  return location.pathname
+}
```

Comment:
Unfortunately this is not that simple. `getCurrentHref()` is part of the embed API (see comment above). Changing this means extending that API.